client/shipping-label/state/actions.js

```diff
@@ -229,22 +233,52 @@ export const updatePackageWeight = ( packageId, value ) => {
 	};
 };
 
-export const confirmPackages = () => ( dispatch, getState, { getRatesURL, storeOptions, nonce } ) => {
+const refreshPreview = ( dispatch, getState, context ) => {
+	if ( ! canPurchase( getState(), context.storeOptions ) ) {
+		return;
+	}
+	const state = getState().shippingLabel;
+	const { form, paperSize } = state;
+	let pckgIndex = 1;
```

Comment:
You can avoid this counter by using the second argument passed to `_.map()` callbacks.